sources/US/SD/wpa-federal-lands.json

```diff
@@ -0,0 +1,18 @@
+{
+    "name" : "SD Waterfowl Production Areas & Federal Lands Open for Hunting",
```

Comment:
Is this a duplicate of data that is already in the public land layer?

Reply:
~90% duplicate. Public Land layer is missing USACOE land along the Missouri River, will open an issue for that.